mlflow/deployments/server/app.py

```diff
@@ -139,6 +154,7 @@ class Config:
                             "name": "gpt-3.5-turbo",
                             "provider": "openai",
                         },
+                        "limit": {"calls": 1, "key": None, "renewal_period": "minute"},
```

Comment:
we might want to remove the key field from the limit model schema of `mlflow.deployments` since it is not used

Reply:
Let's keep if for compatibility with Databricks.